programs/castle-vault/src/adapters/port.rs

```diff
@@ -58,21 +58,21 @@ pub struct PortAccounts<'info> {
     #[account(
         mut,
         seeds = [vault.key().as_ref(), b"port_obligation".as_ref()],
-        bump = port_additional_states.vault_port_obligation_bump
```

Comment:
why remove bump constraints?

Reply:
It's probably my misunderstanding that led to this.
If the bump value is not given, then anchor will search for a valid one using the seeds. But what we want to do here is to actually check the bump value. 
I'll add it back.